### ISSUE TITLE
Is there any interest in supporting older micros?

### ISSUE THREAD

**Issue by gtoal** [aquynh/capstone#1615]
Just wondering if there is anyone in the project considering adding support for 8 bit micros, such as 6502, 6809 and z80?

(this is not a feature request nor an offer to create those, just asking if it is already on anyone's radar)

thanks,

Graham

**Comment by aquynh** [comment]
yes feel free to add support for those architectures - into the "next" branch.

thanks.

**Comment by aladur** [comment]
Current status in next branch for 8-bit micros with the following flavors:
MOS65XX: 6502, 65C02, W65C02, 65816
M680X: 6800, 6801, 6805, 6808, 6809, 6811, 6301, 6309, CPU12, HCS08
